\chapter{Material and Methods}
\label{ch:Mat}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Ethical approval and recruitment of study participants}
Sample recruitment for the two different phenotypes and the healthy volunteers were conducted under different ethics.

\subsection{Psoriasis patient recruitment}

Patient blood samples and normal or psoriatic skin biopsies were collected in collaboration with the Dermatology Department research nurses at the Churchill Hospital, Oxford University Hospitals NHS Trust and Professor Graham Ogg at the Weatherall Institute of Molecular Medicine, University of Oxford under approval from the Oxfordshire Research Ethics Committee (REC 09/H0606/71 and 08/H0604/129). After written informed consent, up to 60 mL of blood from eligible psoriasis patients were collected into 10 mL anticoagulant EDTA-containing blood tubes (Vacutainer System, Becton Dickson).

Psoriasis patients were eligible for recruitment when meeting the following criteria:
\begin{itemize}
  \item over 18 years old
  \item previously or newly diagnose, in a flare and going into biologic therapy for the first time % check with Graham
	\item fulfillment of the clinically accepted Psoriasis Area and Severity Index (PASI) classification for psoriasis diagnosis \parencite{Fredriksson1978}
	\item moderate to severe disease (PASI>5) % check with Graham
	\item less than 2 weeks without antibiotics unless used for prophylaxis % check with Graham or research nurses
	\item available clinical information and written consent
\end{itemize}

Detailed clinical information of the psoriasis cohort is included in %(Chapter \ref{ch:})(Table \ref{tab:}).

\subsection{PsA patient recruitment}
Sample recruitment was performed as part of the Immune Function in Inflammatory Arthritis (IFIA) study established in 2006 (REC/06/Q1606/139)in collaboration with research nurses at the Nuffield Orthopaedic Centre, Oxford University Hospitals NHS Trust and Dr Hussein Al-Mossawi at the Botnar Research Centre. Following informed written consent, blood (30mL) and synovial fluid aspirate (variable upon disease severity) were recruited into 10mL anticoagulant sodium heparin coated tubes (Vacutainer System, Becton Dickson).

Eligibility of the PsA patients was upon fulfillment of the following criteria:
\begin{itemize}
  \item over 18 years old
  \item previously or newly diagnose, with concomitant psoriasis, in a flare and going into biologic therapy for the first time % check with Hussein
	\item fulfillment of the clinically accepted PsA Response Criteria (PsARC) including a physician global assessment questionnaire \parencite{Philipp2011,Clegg1996}
	\item oligoarticular phenotype and na\"{i}ve for any treatment
	\item less than 2 weeks without antibiotics unless used for prophylaxis % check with Hussein
	\item available clinical information and written consent
\end{itemize}

Further details about the cohort and clinical information can be found in (Chapter \ref{ch:})(Table \ref{tab:}).

\subsection{Healthy volunteer recruitment}
Recruitment of healthy volunteers was conducted as part of the study Genetic Diversity and Gene Expression in White Blood Cells with approval from the Oxford Research Ethics Committee (REC 06/Q1605/55). Up to 80mL of blood were collected into 10mL anticoagulant EDTA-containing blood tubes, similarly to the psoriasis sample recruitment.

The criteria for healthy individuals to participate in the study was:
\begin{itemize}
  \item over 18 years old and preferably British or European
  \item no family history of psoriasis, PsA, RA or SpA
	\item matched sex and age with the psoriasis cohort
	\item less than 2 weeks since last infectious process
	\item available clinical information and written consent
\end{itemize}


\section{Sample processing}
\label{sample_processing}
Blood, synovial fluid and skin biopsies were processed straight after recruitment following the appropriate protocols.

\subsection{PBMC and synovial fluid cells isolation}
PBMC were isolated from blood samples through density gradient separation using Ficoll-Paque. Total synovial fluid (SF) cells (SFC) were isolated by centrifugation at 500g for 5 min. Both were washed twice in Hank’s balanced salt solution without calcium or magnesium (Thermo Fisher Scientific) and resuspended in phosphate saline buffer (PBS, Gibco) supplemented with 0.5\% fetal calf serum (FCS, Invitrogen) and 2mM ethylenediaminetetraacetic acid (EDTA, Sigma)prior to cell types separation. Cell numbers and viability were determined by manual count using a haemocytometer and trypan blue (Sigma).

\subsection{Primary cell isolation using magnetic-activated cell sorting}
For the work related to psoriasis and healthy volunteers, primary cell subpopulations were separated using magnetic-activated cell sorting (MACS, Miltenyi). Positive selection was performed for consecutive isolation of CD19$^{+}$ B cells, CD8$^{+}$ T cells, CD14$^{+}$  monocytes and CD4$^{+}$ T cells with AutoMACS Pro (Miltenyi) and cells were manually counted as previously described. MACS separation was chosen over Fluorescence-associated cell sorting (FACS) due to time and logistic constrains in the sample processing and therefore cell numbers in down stream application may not be as exact.

\subsection{Primary cell isolation using fluorescence-activated cell sorting}
Primary cell subpopulations from controls to study the effect of cryopreservation in chromatin states (Chapter 3) or PsA blood and SF samples were isolated by FACS. PBMC and SFC were resuspended in PBS 1mM EDTA (FACS buffer) at 10x10$^6$ cells/mL, stained with the appropriate antibody cocktail (Table \ref{tab:FACS_antibodies}) for 30 min at 4{$^\circ$}C, washed with FACS buffer and centrifuged at 500g for 5 min at 4{$^\circ$}C. For the samples used in Chapter 3, a modified FACS buffer supplemented with 3 mM EDTA , 2\% FCS and 25mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES, Invitrogen) was used to avoid cell clumping after cryopreservation and short recovery.After removing the supernatant, cells were resuspended in FACS buffer prior to separation. 

In the controls samples of Chapter 3 only CD14$^{+}$ monocytes and CD3$^+$ CD14$^{-}$ CD4$^{+}$ T cells were isolated in the SONY SH800 cell sorter. For the PsA samples, separation of  CD19$^{+}$ B cells, memory T cells (CD3$^{+}$ CD14$^{-}$ CD4$^{+}$ CD45RA$^{-}$ and CD3$^{+}$ CD14$^{-}$ CD8$^{+}$ CD45RA$^{-}$) ,CD14$^{-}$ monocytes and CD56$^{-}$ NK was performed using FACS Aria (BD) cell sorter from both PBMC and SFC. Bulk sorted cells were collected in 1.5mL tubes in PBS 1\% FCS, %whilst single cell and small bulk sorting was performed in 96-well plates in the appropriate buffer (See RNA-seq section). 
Different nozzle sizes were chosen for bulk and single-cell sorting and OneComp eBeads (eBioscience) were used for compensation of fluorescence spill over.


\begin{table}[htbp]
%\setlength{\tabcolsep}{20pt} only to stretch the columns if you want
%\renewcommand{\arraystretch}{1.5}
\begin{tabular}{@{} c c c c c}
\toprule
\textbf{Surface} & \textbf{Fluorochrome} & \textbf{Manufacturer} & \textbf{Clone} & \textbf{Dilution} \\
\textbf{marker} & \textbf{PsA/CTL} & \textbf{PsA/CTL} & \textbf{PsA/CTL} & \textbf{PsA/CTL} \\
\midrule
\midrule
Viability & eFluor780 & - & eBioscience & 1:500\\
CD3 & FITC/AF700 & SK7/UCHT1 & BioLegend & 1:50/1:50\\
CD4 & APC & RPA-T4/RPA-T4 & BioLegend & 1:50/1:50\\
CD8a & PE & RPA-T8 & BioLegend & 1:100\\
CD45RA & BV421 & HI100 & BioLegend & 1:25\\
CD19 & PerCP-Cy5.5 & SJ25C1 & BioLegend & 1:50\\
CD14 & Pe-Cy7/FITC & M5E2/TUK4 & BioLegend/Miltenyi & 1:50/1:100\\
CD56 & BV510 & HCD56 & Biolegend & 1:25 \\
\bottomrule
\end{tabular}
\medskip %gap
\caption[Antibody panel used for FACS separation of primary cell populations in controls and PsA samples.]{\textbf{Antibody panel used for FACS separation of primary cell populations in controls and PsA samples.}Details regarding target molecule, fluorochrome, clone, supplier and dilution used for PBMCs and SFMCs staining are provided for each of the antibodies in the panel. In the controls only CD3, CD4 and CD4 markers were used.}
\label{tab:FACS_antibodies}
\end{table}
\bigskip %bigger space



%Long table example
%\begin{longtable}{ p{.15\textwidth} p{.25\textwidth} p{.2\textwidth} p{.15\textwidth} p{.15\textwidth}}
%\caption[Antibody panel used for FACS separation of primary cell populations in controls and PsA samples]{\textbf{Details regarding target molecule, fluorochrome, clone, supplier and dilution used for PBMC and SFC staining are provided for each of the antibodies in the panel. In controls only CD3, CD4 and CD4 markers were used.}}
%\label{tab:FACS_antibodies} \\
%
%\toprule
%\textbf{Surface marker} & \textbf{Fluorochrome PsA/controls} & \textbf{Manufacturer PsA/controls} & \textbf{Clone PsA/controls} & \textbf{Dilution PsA/controls} \\
%\midrule
%\midrule
%Viability & eFluor780 & - & eBioscience & 1:250\\
%CD3 & FITC//AF700 & SK7/UCHT1 & BioLegend & xxx/1:50\\
%CD4 & APC & RPA-T4(RPA-T4) & BioLegend & 1:50/1:50\\
%CD8a & PE & RPA-T8 & BioLegend & xxx\\
%CD45RA & BV421 & HI100 & BioLegend & xxx\\
%CD19 & PerCP-Cy5.5 & SJ25C1 & BioLegend & xxx\\
%CD14 & Pe-Cy7/FITC & M5E2/TUK4 & BioLegend/ Miltenyi & xxx/1:100\\
%CD56 & BV510 & NCAM16.2 & BD & xxx\\
%\bottomrule
%\medskip %gap
%
%\end{longtable}
%\bigskip %bigger space
%




\subsection{Skin biopsies processing and adherent assay}
KC enrichment from skin biopsies was performed as described in Gutowska-Owsiak and colleagues \parencite{Gutowska‐Owsiak2012}. Skin biopsies (approximately 4mm) were washed with PBS, cut in 1mm width strips and incubated in 2U/mL of dispase II (Sigma) overnight at 4{$^\circ$}C. The epidermis was separated from the dermis and either snap-frozen in liquid nitrogen (for RNA extraction) or further digested in trypsin (Invitrogen) at 37{$^\circ$}C for 5 min, when used for chromatin accessibility assay. After digestion the resulting cell suspension was filtered through a 70$\micro$m nylon strainer (BD) and washed with PBS. In some instances cells were manually counted and aliquoted for ATAC-seq processing. In others, cell from each of the biopsies were resuspended in KGM-2 BulletKit (Lonza) supplemented with 0.06mM Ca$^2{+}$ and cultured in a collagen IV coated 96-well plate at 4{$^\circ$}C for 10 min or 3 hours, upon experimental requirements (see Chapter X). After culturing, cells were washed twice with 200$\micro$L of PBS and kept at 37{$^\circ$}C for downstream processing.



\section{Experimental protocols}
\subsection{Cryopreservation and cell culture}
For the controls samples in Chapter 3, 40-50x10$^6$ of PBMC were freeze-thawing using a modified version of the \parencite{Kent2009} protocol, where cells were pre-conditioned in RPMI 1640 (brand) complete medium supplemented with2 mM L-glutamine, 100U penicillin and strep 100$\micro$g/mL and 50\% FCS for 30 minutes and afterwards diluted 1 in 2 in complete RPMI 1640 (supplemented as previously described) with 20\% dimethyl sulfoxide (DMSO, Sigma). PBMC underwent slow cryopreservation at -80{$^\circ$}C in isopropanol at -1{$^\circ$}C per minute and stored for a minimum of two weeks in liquid nitrogen. PBMC were thaw, resuspended in supplemented complete RPMI 1640 with 10\% FCS at a density of 10$^6$ cells/mL and rested for 30 min at 37°C, 5\% CO2 in 25mL non-adherent polypropylene cell culture flasks followed by filtering through a 40$\micro$m to obtain an homogenous cell suspension undergoing FACS separation.
%
Frozen Normal human epidermal keratinocytes (NHEK) in passage 3 were recovered and cultured at a cell density of 5x10$^6$ cells/mL in a 75 mL adherent cell culture flask (brand) in EpiLife basal medium (Gibco) following manufacturer's instructions. After recovery NHEK were trypsinised at room temperature for 8 minutes followed by trypsin inactivation with EpiLife 10\% FCS, centrifugation at 180g for 10 min at room temperature and manual counting with trypan blue. NHEK were seeded in a 96-well plate in 100uL of medium at a cell density of 160 cells/$\micro$L. NHEK were cultured for 2 days to a 90-100\% confluence before being used downstream.

\subsection{ATAC-seq, Fast-ATAC and Omni-ATAC}
Improved versions of the ATAC-seq protocol were progressively used in the thesis for assessment of chromatin accessibility in different primary cells, including CD14$^{+}$ monocytes, CD4$^+$ and CD8$^+$ T cells, CD19$^+$ B cells and CD56$^+$ NK cells. The subsequent version aimed to reduce the amount of mitochondrial DNA and improve the ratio of signal to noise for this technique.

After MACS separation, primary cells were manually counted as above specified and they were resuspended in PBS with 1\% FCS. As previously stated, due to reduced accuracy of manual cell counting compared to FACS sorting, in my experiments ATAC-seq was performed using an estimated number of cells between 50,000 to 100,000. ATAC-seq was performed as described in Buenrostro \textit{et al.}, 2013 with minor modifications. Cells were centrifuged at 500g for 5 min at 4{$^\circ$}C. After removing the supernatant cells were lysed for 10 min, the nuclei were transposed using the Nextera Tn5 transposase (Illumina) for 40 min at 37{$^\circ$}C and DNA was purified using the PCR MinElute kit (Qigen). Additional modifications and performance in 96-well plates were implemented for KC and they will be described in %(Chapter \ref{ch:}). 

After appropriate determination of the amount of DNA amplification using qPCR, samples were amplified and singled indexed for 11 PCR cycles using modified Nextera primers from Buenrostro \textit{et al.},2013 (Table\ref{tab:Indexing_primers}). The resulting DNA libraries were purified using the MinElute PCR purification kit  (Qiagen) and additional Agencourt AMPure XP Magentic Beads (Beckman Coulter), according to the manual specifications, to remove the remaining adaptors and primer dimers.

\begin{landscape}
\begin{table}[htbp]
\setlength{\tabcolsep}{20pt}
%\renewcommand{\arraystretch}{1.5} makes it longer
\begin{center}
\begin{tabular}{@{} c c}
\toprule
\textbf{Primer name} & \textbf{Full sequence} \\
\midrule
\midrule
Ad1.noMX & AATGATACGGCGACCACCGAGATCTACACTCGTCGGCAGCGTCAGATGTG \\
Ad2.1 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{TCGCCTTA}GTCTCGTGGGCTCGGAGATGT \\
Ad2.2 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{CTAGTACG}GTCTCGTGGGCTCGGAGATGT \\
Ad2.3 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{TTCTGCCT}GTCTCGTGGGCTCGGAGATGT \\
Ad2.4 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{GCTCAGGA}GTCTCGTGGGCTCGGAGATGT \\
Ad2.5 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{AGGAGTCC}GTCTCGTGGGCTCGGAGATGT \\
Ad2.6 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{CATGCCTA}GTCTCGTGGGCTCGGAGATGT \\
Ad2.7 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{GTAGAGAG}GTCTCGTGGGCTCGGAGATGT \\
Ad2.8 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{CCTCTCTG}GTCTCGTGGGCTCGGAGATGT\\ 
Ad2.9 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{AGCGTAGC}GTCTCGTGGGCTCGGAGATGT \\
Ad2.10 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{CAGCCTCG}GTCTCGTGGGCTCGGAGATGT \\
Ad2.11 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{TGCCTCTT}GTCTCGTGGGCTCGGAGATGT \\
Ad2.12 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{TCCTCTAC}GTCTCGTGGGCTCGGAGATGT \\
Ad2.13 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{ATCACGAC}GTCTCGTGGGCTCGGAGATGT \\
Ad2.14 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{ACAGTGGT}GTCTCGTGGGCTCGGAGATGT \\ 
Ad2.15 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{CAGATCCA}GTCTCGTGGGCTCGGAGATGT \\
Ad2.16 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{ACAAACGG}GTCTCGTGGGCTCGGAGATGT \\
Ad2.22 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{TGTGACCA}GTCTCGTGGGCTCGGAGATGT \\
Ad2.23 & CAAGCAGAAGACGGCATACGAGAT\textcolor{blue}{AGGGTCAA}GTCTCGTGGGCTCGGAGATGT \\
\bottomrule
\end{tabular}
\medskip %gap
\caption[Modified Illumina Nextera indexing primers]{\textbf{Name and full sequence of the PCR primers used for amplification, indexing and pooling of the ATAC-seq and ChIPm samples in this thesis. These primers were designed by Buenrostro \textit{et al.}, 2013 and they are an modified version of the Nextera Illumina primers optimised for larger molecular weight DNA fragments from low input samples. All samples were indexed with the universal primer Ad1.noMx and one of the additional 18 primers. The indexing sequence of each of the primers is in blue text.}}
\label{tab:Indexing_primers}
\end{center}
\end{table}
\end{landscape}
\bigskip %bigger space

Following the Nature Methods publication of Corces et al., 2016, the  initial ATAC-seq protocol was replaced by a modified version named Fast-ATAC. It was specifically optimised for hematopoietic cells and combined cell lysis and transposition in a single step. Fast-ATAC was performed as described by Corces et al., 2016 with minor modifications. Since 5,000 cells was considered the lower limit to generate good quality data in Fast-ATAC, in my experiments I used 20,000 MACS or FACS sorted cells, to account for inaccurate manual cell counting as well as possible cell loss over centrifugation steps. The Fast-ATAC reaction was performed for 30 min at 37{$^\circ$}C and agitation at 400rpm. DNA was purified as in ATAC-seq and libraries were generated following 13 cycles of PCR amplification, following appropriate cell cycle determination. Purification following PCR were performed using Agencourt AMPure XP Magentic Beads only.

Omni-ATAC, a third generation of ATAC-seq was published by Corces et al., 2017. It consisted in an universal protocol with individual cell lysis and transposition reactions intercalated with a washing step, to remove mitochondrial DNA and other cell debri %check. 
Omni-ATAC was performed as described by Corces and colleagues \parencite{Corces2017} using 50,000 cells.

Following either of the three protocols, DNA tagmentation profiles were assessed with the D1000 high sensitivity DNA tape (Agilent) as part of the quality control and quantified using the Kapa kit (Roche), following the manufacturer's instructions. Pools of 12 to 16 libraries were sequenced in one to 3 lanes of the HiSeq4000 Illumina platform by the Oxford Genomics Centre at the Wellcome Trust Centre for Human Genetics to achieve approximately 50 million paired-end reads.


\subsection{Chromatin immunoprecipitation with sequencing library preparation by Tn5 transposase}
Assessment of the histone mark H3K27ac (active enhancer and promoter marker) in the chromatin of psoriasis patients from Cohort 1B and four age matched healthy volunteers was performed using a low cell input Chromatin Immunoprecipitation (ChIP) method know as ChIPmentation (ChIPm). For each individual and cell type three histone marks, including H3K27ac, H3K4me1 and X were tested in chromatin isolated from 100,000 cells and compared to an input chromatin sample processed in parallel. Samples were processed following the protocol published by Schmidl and colleagues \parencite{Schmidl2015} with some modifications. Aliquots of 600,000 cells of MACS sorted cell types, as described in \ref{sample_processing}, were fixed with 1\% formaldehyde (Sigma) and snap frozen in dry ice and ethanol prior to storage at -80{$^\circ$}C. Chromatin sonications of the different individuals and cell types were performed in one batch using Covaris M220(Covaris). Each of the aliquots was resuspended in 130$\micro$L of SDS lysis buffer (Table\ref{tab:ChIPm_buffers}), sonicated for 8 min using a duty factor of 5\% and aliquoted for single ChIPm reactions prior to long term storage at -80{$^\circ$}C.

Sonicated chromatin aliquots were thawed and 1.5 volumes of ChIP equilibration buffer (Table\ref{tab:ChIPm_buffers}) was added in order to  reduce the SDS concentration to 0.1\% and to achieve the appropriate concentration of NaCl and Triton-X100. For the immunoprecipitation step, samples were incubated with 1$\micro$g of the Diagenode antibody (C15410196) overnight in rotation at 4{$^\circ$}C . Protein-A Dynabeads (Invitrogen) were also washed in beads wash buffer (Table\ref{tab:ChIPm_buffers}), blocked with yeast tRNA (Ambion) and BSA (NEB) overnight in rotation at 4{$^\circ$}C and added to the sample-antibody mix for incubation. After appropriate washes, tagmentation was performed on the bead-antibody bound crosslinked complexes, which prevents overtagmentation of the DNA. This was followed by protein K digest, reverse crosslinking, elution of the DNA and from the beads and purification using PCR MinElute kit. The chromatin input control samples were quantified with Qbit after reverse crosslinking and 1ng of chromatin was used for tagmentation.

Amplification by qPCR was done in each of the samples and control inputs to identify the number of full cycles required to reach one-third of the final fluorescence. Libraries were amplified for the number of cycles minus one determined with this strategy to minimise the total number of PCR replicates. The primers used for amplification and indexing were the ones optimised by Buenrostro and colleagues (Table\ref{tab:Indexing_primers}). The number of amplification cycles for each of the samples is recorded in %\ref{Table:}.

\begin{table}[htbp]
\begin{tabular}{@{} c c}
\toprule
\textbf{Reagent} & \textbf{Final concentration}\\
 & & \\
\bottomrule
 & \textbf{SDS lysis buffer} & \\
\midrule
\midrule
SDS & 0.25\% & Sigma \\	
EDTA	& 1mM & X \\
Tris-HCl pH 8 & 10mM & Sigma \\
PI & 1X & Roche \\
Water & - \\
\bottomrule
 & \textbf{ChIP equilibration buffer}  & \\
\midrule
\midrule
Triton-X100 & 1.66\% \\
EDTA	& 1mM \\
NaCl	& 233mM \\
Tris-HCl pH 8 & 10mM \\
PI & 1X \\
Water & - \\
\bottomrule
 & \textbf{Beads washing buffer} & \\
\midrule
\midrule
SDS & 0.1\% \\
EDTA	& 50mM \\
NaCl & 150mM \\
NP-40 & 1\% \\
Tris-HCl pH 8 & 10mM \\
PI & 1X \\
Water & - \\
\bottomrule

 & \textbf{ChIP buffer} & \\
\midrule
\midrule
SDS & 0.1\% \\
Triton-X100 & 1\% \\
EDTA	& 1mM \\
NaCl & 140mM \\
Tris-HCl pH 8 & 10mM \\
PI & 1X \\
Water & - \\
\bottomrule
\end{tabular}
\medskip %gap
\caption[ChIPm buffers modified from Schmidl \textit{et. al}, 2015]{\textbf{Composition of the three modified buffers in house for the ChIPm protocol: SDS lysis buffer, ChIP equilibration buffer, beads washing buffer and ChIP buffer. For each of the buffers the reagents, composition and supplier are indicated.The final volume prepared for each buffer was adjusted depending on the number of samples processed at the time. Sodium dodecyl sulfate (SDS), PI (proteinase inhibitor). Supplier for each of the reagents as follows: SDS (Sigma), EDTA(xxx), Tris-HCl pH8 (xx), Triton-X100 (xxx), NP-40 (Sigma) NaCl(xx), PI (Roche) and water (Ambion) }}
\label{tab:ChIPm_buffers}
\end{table}
\bigskip %bigger space

% Only doing ChIP for one histone mark so table not needed. Integrate in the text
%\begin{table}[htbp]
%\setlength{\tabcolsep}{20pt}
%\renewcommand{\arraystretch}{1.5}
%\begin{tabular}{@{} c c c c}
%\toprule
%\textbf{Histone mark} & \textbf{Feature} &\textbf{$\micro$g per sample} & \textbf{Manufacturer}\\
%\midrule
%H3K27ac & Active enhancer, promoter & 2 & Diagenode (C15410196)\\
%H3K4me1 & Enhancer & 1 & Diagenode (C15410194)\\
%H3K4me3 & Active promoter, enhancer & 1 & Diagenode (C15410003) \\
%\bottomrule
%\end{tabular}
%\medskip %gap
%\caption[Antibody panel used for immunoprecipitation of histone marks in ChIPm]{\textbf{Details regarding the histone marks, the the most likely chromatin state delineated, the amount of antibody required per reaction and the supplier and catalog num of the antibodies.}}
%\label{tab:ChIPm_antibodies}
%\end{table}
%\bigskip %bigger space


\subsection{RNA extraction and gene expression quantification}

\subsubsection{RNA extraction}
Following MACS isolation of the different cell types from the psoriasis and matched healthy controls, between 2-3x10^6 cells were resuspended in 350$\micro$L of RNAProtect (Qiagen) or RLT buffer (Qiagen) supplemented with 0.1\% of beta-mercaptoethanol (BM, Sigma) and snap frozen in dry ice before storage at -80{$^\circ$}C. Cells isolated from psoriasis and control Cohort 1A (Chapter \ref{ch:}) were preserved in RNAProtect, which stops any biochemical reaction and transcriptional activity maintaining cell integrity. At early stages of the project, when I was uncertain of time frames to process the different material from the acquires samples, I decided to use RNAProtect to preserve cells for future RNA extraction to guarantee high quality in case storage exceeded 6 months. In the psoriasis and control samples from Cohort 1B %(Chapter \ref{ch:}) 
, cells were resuspended in 0.1\% BM supplemented RLT buffer, which lysates cells and prevents RNA degradation. Cell lysates were homogonised using the QIAshredder (Qiagen) prior to RNA extraction. When starting from RNAProtect preserved material, cells were centrifuged at 300g for 10 min at room temperature, the supernatant were removed and the pellets were resuspended in 350$\micro$L of RLT 0.1\% BM buffer, prior to homogenisation with QIAshredder.

Total RNA were extracted using the AllPrep DNA/mRNA/microRNA Universal kit (Qiagen) following the manufacturer's instructions. RNA extractions were performed in batches of 12 samples, including all cell types from each individual and a balanced numbers of psoriasis and control samples, to minimise batch effect correlation with phenotype groups. Basic quantification was performed with NanoDrop (Thermo Scientific) before storage at -80{$^\circ$}C.


\subsubsection{RNA-seq}
RNA-seq quality control (QC), quantification, library preparation and sequencing were carried out by Oxford Genomics Centre at the Wellcome Trust Centre for Human Genetics in two independent batches of samples, each including Cohort 1A or Cohort 1B, respectively. Processing of samples in two batches was due to logistics of patients recruitment in the project. Quality control and quantification were assessed with the Bioanalyzer (Agilent). Samples were depleted from ribosomal RNA using Ribo-Zero rRNA Removal kit (Illumina) prior to library preparation using TruSeq Stranded Total RNA (Illumina). Unlike strategies using polyadenylated transcripts selection, this method allowed to preserve non-polyadenylated transcripts including nascent pre-mRNA (unspliced) and functionally relevant lncRNAs. For each of the cohorts, all libraries were pooled together and  sequenced over several lanes of HiSeq4000 to a depth of 50 million total reads per sample, in order to maintain an appropriate level of sensitivity for subsequent expression analysis given the greater complexity of these libraries.

\subsubsection{Gene expression quantification by qPCR array}
Expression of immune-relevant genes was profiled by qPCR using the RT2 Profiler PCR Array (PAHS-3803Z, Qiagen) in collaboration with UCB. This platform included primers to test expression for 370 key genes involved in immune response during autoimmunity and inflammation as well as appropriate house-keeping genes. In brief, RNA was extracted, as detailed previously, from CD14$^+$ monocytes, mCD4$^+$ and mCD8$^+$ cells isolated from PsA patients by FACS. RNA was also extracted for the same cell types isolated from PB in three healthy individuals. Retro-transcription for cDNA synthesis was performed by UCB and subsequently used for qPCR gene expression quantification following the PCR array's manufacturers instructions.  

\subsubsection{Single-cell RNA-seq} 
\label{scRNA_processing}
scRNA-seq data was generated using 10X Genomics technology Chromium single cell 3' expression library prep kit (PN-120267) by the Oxford Genomics Centre at the Wellcome Trust Centre for Human Genetics. In short, approximately 3,000 cells from the PBMCs or SFMCs cell suspensions from each of the three PsA patients (Chapter \ref{ch:Results2}) were partitioned into single-cell gel beads in emulsion (GEMs) using the 10X Chromium controller system. Retro-transcription for cDNA synthesis was performed within the GEMs, which included a 16bp 10x barcode, a 10bp unique molecular identifier (UMI) and poly-dT primers. The cDNA was released from the GEMs, followed by PCR amplification, enzymatic fragmentation and size selection. Afterward, appropriate sequencing Illumina indexes were incorporated into the samples through library preparation. Sequencing was performed using PE HiSeq4000 with 26pb for read 1 and 98bp for read 2 at a depth of approximately 50,000 reads per cell, following standard 10X library sequencing requirements. 




%\subsubsection{Small-bulk RNA-seq } 
%Between 100 to 500 cells of the five populations isolated from PsA patients were FACS sorted into 2$\micro$L of cell lysis buffer and processed for library prep as in Picelli \textit{et al.}, 2014 by the Oxford Genomics Centre at the Wellcome Trust Centre for Human Genetics. Libraries were sequenced with Illumina HiSeq4000 xxxx bp single-end at a depth of XXX reads per cell.


%\subsection{Single-cell analysis of the V(D)J T cell receptor repertoire}
%Single-cell sequencing of the V(D)J segments from TCR transcripts was performed simultaneously with the 10X Genomics technology Chromium 5' expression library kit (PN-1000014) by the Oxford Genomics Centre at the Wellcome Trust Centre for Human Genetics. In short, full-length cDNA was amplified by PCR with primers against the 5’ and 3’ ends of the barcode sequences inherent to the 10X Genomics bead technology. The amplified material was divided for use in 5' total scRNA-seq (as specified previously in \ref{scRNA_processing}) and also for enrichment of the TCR by PCR amplification with specific primers. TCR enriched cDNA was followed by enzymatic fragmentation and size selection in order to obtain variable length fragments spanning the V(D)J segments. Library prep and indexing was followed by Illumina HiSeq4000 xxxx bp single-end sequencing at a depth of XXX reads per cell.

\subsection{DNA extraction and SNP genotyping}
DNA isolation was performed using the AllPrep DNA/mRNA/microRNA Universal kit (Qiagen) following the manufacturer's instructions. Basic quantification was performed using NanoDrop (Thermo Scientific) and kept at -80{$^\circ$}C for long term storage. For each of the patients and controls, the extracted DNA was amplify by PCR using forward (5'-CACTGTGGAGGGAGGAACAA-3') and reverse (5'-CGTGTTGGCCAGGATAGTCT-3')primers annealing up and down stream the SNP rs4672505, respectively. The 390 bp PCR product was purified using MinElute PCR purification kit, quantified by dsDNA Qbit kit (Invitrogen) according to the manufacturer's instructions and prepared for Sanger sequencing using the Mix2Seq service (Eurofins). The forward and reverse sequences were analysed with BioEdit software.



\subsection{Mass cytometry using cytometry by time of flight (CyTOF)}
Mass cytometry assay was performed by Dr Nicole Yager in collaboration with UCB following their in-house protocol for CyTOF instrument. Briefly, whole blood and SF from three PsA patients were fixed for 5 min with 1.6\% PFA within 30 min of venipuncture/aspiration, respectively. These samples are defined as time 0h. In addition, whole blood and SF were incubated at 37{$^\circ$}C for 6 h in the presence of the protein transport inhibitors 1X BD GolgiStop (monensin) and 1X BD GolgiPlug (brefeldin A). After 5 h 45 mins the samples were treated with cisplatin to facilitate discrimination of dead cells, and then fixed as previously described.  These samples are defined as time 6h. After appropriate treatment, red blood cells were lysed and cell suspensions were washed with PBS and stained with Abs against the cell surface markers of the intra-cellular staining (ICS) panel.  The samples were further permeabilised and stained with antibodies against intracellular targets (Table \ref{tab:CyTOF}). Treatment with monensin and brefeldin A prevented the release of cytokines from the cells and allowed measuring the intrinsic cytokine production rate in basal conditions. 


\begin{table}[htbp]
\setlength{\tabcolsep}{20pt}
\renewcommand{\arraystretch}{1.5}
\begin{tabular}{@{} c}
\toprule
\textbf{ICS panel} \\
\midrule
CD248, CD19, GP38, FAP, CD8a, IL8, CD16, CD25, CD123, IL-21, FceR, 
IL-17F, IL-17A, IL-10, CD11c, CD14, IL6, IFN-$\gamma$, GM-CSF, FCeR, CD3, 
CD45RO, CD56, HLA-DR, IL-13, CD117, CD4, IL4, IL-2, TNF$\alpha$,CD161, CD-11b, CD45 \\
\bottomrule
\end{tabular}
\medskip %gap
\caption[Molecules targeted by the mass cytometry ICS staining panel in whole blood and SF.]{\textbf{Molecules targeted by the mass cytometry ICS staining panel in whole blood and SF.} The molecules targeted by the Abs used in the ICS staining panel are listed. Note the panel also included Abs recognising surface markers to identify the populations of interest for further analysis of the intracellular cytokine production.}
\label{tab:CyTOF}
\end{table}
\bigskip %bigger space


\section{Computational and statistical analysis}

\subsection{ATAC-seq data analysis}
\label{ATAC_analysis}
ATAC-seq, Fast-ATAC and Omni-ATAC data were analysed using an in house pipeline towards which development I made an important contribution. The pipeline performs single sample data processing and it also builds a combined master list for each of the comparisons of interest to later perform differential analysis. 

\subsubsection{Next generation sequencing data analysis}
NGS data for each of the samples was trimmed for low quality base pairs and Nextera adapter sequences using cutadapt \parencite{} before general QC assessment using fastqc \parencite{}. Trimmed data was alignment to the reference genome built hg19 using bowtie2 \parencite{Langmead2006} and the following parameters -k 4 -X 2000 -I 38 --mm -1, consistently with other publications \parencite{Buenrostro2013, Corces2016}. Samtools \parencite{} was used to remove PCR duplicates reads previously marked with Picard Tools \parencite{} as well as low MAPQ (${<}$30) non-uniquely and non-properly paired reads. The resulting bam file was additional filtered to remove mitochondrial DNA and reads were adjusted by +4 bp in the plus strand and by -5 bp in the minus strand to represent the center of the transposition binding event. Pileup tracks (bigWig files) for visualisation were generated using bedtools genomeCoverageBed where each value represents number of reads per position \parencite{} and the UCSC genome browser bedGraphToBigWig tool \parencite{}.  

\subsubsection{Peak calling, filtering and sample quality}
Peak calling was performed using MACS2  callpeak \parencite{} and the parametres --nomodel --shift -100 --extsize 200 --p 0.1 --keep-dup all --call-summits and filtered for those overlapping with the blacklisted features from ENCODE project \parencite{}. The --shift and --extsize parameters were set according to the recommendations of MACS2 for DHS and following other ATAC-seq publications \parencite{Buenrostro2015, Corces2016}. The pval cut off for peak filtering was determine for each of the cell types separately. 
For a particular cell type, the bam file of each sample (patients and controls) was partitioned in two (pseudorreplicates) and peak calling was performed in each of them followed by Irreproducibility Discovery Rate (IDR) analysis to assess the percentage of peaks sharing IDR at each corresponding pval. Median of the pval showing the greater percentage across all the samples was used to filter each peak list. Peak summits were replaced by the median of the summits for those with multiple called summits and extended +/- 250bp to create a non-overlapping homogenous 500bp peak list \parencite{Buenrostro2015, Corces2016}. 

Sample quality was determined by the fol-change enrichment of ATAC-seq signal across all the TSS identified by Ensembl, since chromatin is expected to be more accessible at the sites of transcriptional initiation compared to the flanking regions. Fraction of reads in peaks (FRiP)was calculated for samples in Chapter \ref{ch:Results1} as the overlap between the peak list filtered for FDR$<$0.01 and with bedtool and the fragment files in bed format using bedtools intersect --f 0.1.

\subsubsection{Combined peak master list and differential analysis}
To perform differential open chromatin analysis a non-overlapping 500bp peak master list was built for each cell type separately by union of all the peaks that were present in at least 30\% of the samples, regardless patient and control group. Reads overlapping each of those peaks were retrieved for each sample using HTSeq-count algorithm \parencite{} to build a combined count matrix. An empirical 95\% confidence cut-off for the number of counts in absent peaks was calculated on the raw count matrix and used to filter out some peaks in the master list before proceeding to differential analysis (\parencite{Xinmin2005,Jonker2014}). Differential analysis was performed using DESeq2 v1.20 R packaged taking into account paired samples for the PsA analysis or correcting for covariates for the case-control psoriasis analysis (which covariates???)\parencite{Love2014}.
A combined master list for all cell types and tissues (when applicable) was also built following the same strategy, normalised and used for principal component analysis (PCA). 



\subsection{ChIPm data analysis}

\subsubsection{Next generation sequencing data analysis}
ChIPm NGS data from samples and inputs were processed similarly to ATAC-seq (\ref(ATAC_analysis)) for trimming, mapping and filtering with minor modifications. Particularly, MAPQ30 threshold filtering score was lowered to 10 instead of 30. For visualisation of each sample, biWig files with subtracted noise from the input were generated using MACS2 bdgcmp -m subtract and bedGraphToBigWig tools .

\subsubsection{Peak calling, filtering and sample quality}
Peak calling for each sample was performed accounting for the background in the input samples using MACS2 callpeak --bw 200 --p 0.1 --keep-dup all --call-summits. In this case the average library fragment size(--bw) was used by MACS2 to first empirically find the model that better represent the precise protein-DNA interaction and then calling peaks with statistical confidence (pval) using the input sample
to calculate the local background. Filtering and down stream peak homogenisation was performed similarly to ATAC-seq analysis (\ref(ATAC_analysis)). For QC purposes the peak list were filtered using FDR$<$0.05.

Sample quality was determined by the combination of a series of measurements. FOr library complexity the non-redundant fraction (NRF) and PCR bottleneck coefficients (PBC1 and PBC2) were calculated following ENCODE guidelines (https://www.encodeproject.org/chip-seq/histone/). Signal-to-noise was evaluated based enrichment of the H3K27ac reads across all the TSS identified by Ensembl and the FRiP, calculated as previously specified. FRiP was only taken into account in the samples passing the NFR and PBC1/PBC2 filtering.

\subsubsection{Combined peak master list and differential analysis}
To perform differential histone modification analysis a peak master list was built and counts at each of the locations were performed as described for ATAC-seq (\ref(ATAC_analysis)). PCA analysis was performed for samples from all cell types and individuals in a combined master list.

Differential analysis



\subsection{RNA-seq data analysis}

\subsubsection{Bulk RNA-seq analysis}
The ribo-depleted RNA-seq data generated was mapped using the aligner STAR \parencite{Dobin2013} against the Gencode hg19 annotation file containing reference chromosomes, scaffolds and assembly patches. The annotation file comprised 2,840,278 gene entities, including lnc-RNAs. Mapping allowed multiple alignments and only retained those with the best score and a miss-match percentage lower than 0.04\%. Duplicates were marked and removed using Picard Tools. The filtered de-duplicated and sorted bam files were used to retrieve counts at each of the genes location of the annotation file using HTSeq-count. Differential gene expression analysis was performed with DESeq2 v1.20 R package filtering out those genes with five or less reads in at least eight samples (smallest group size corresponding to the psoriasis patients' samples). Independent filtering of genes with unreliable expression levels to reduce multiple testing, outliers removal using Cook's distance and moderation of log$_2$FC were enabled when using DESeq2. Differentially expressed transcripts were filtered based on FDR$<$0.05 or FDR$<0.01$, depending on the downstream analysis. Batch effect was included as covariate in in the DESeq2 model to perform the DGE analysis between psoriasis patients and healthy controls for the four cell types of interest isolated from PBMCs. This effect related to the RNA extraction, library preparation and sequencing of the cohort 1A and cohort 1B samples from psoriasis study (including healthy controls), which were conducted in two different batches.Lnc-RNAs were annotated using the list provided by gencode.v19 (https://www.gencodegenes.org/releases/19.html). The design of the psoriasis skin DGE analysis consisting of paired lesional and uninvolved skin biopsies obtained from each patient was taken into account by the DESeq2 model to increase the power of the analysis.



\subsubsection{Single-cell RNA-seq analysis}
scRNA-seq generated with 10X for bulk PBMCs and SFMCs from three PsA patients (see \ref{ch:Results3}) were processed using Seurat R package \parencite()
PBMCs and SFMCs individual libraries filtered for less than 7.5\% MT and 500 to mean nGenes+1SD, approximately 1,800 genes per cell across the 6 libraries,




\subsection{Genomic region annotation, pathway enrichment analysis and visualisation}
Annotation of genomic regions and signalling-pathways visualisation were performed with two functionalities of the R package and web-app Atlas and Analysis of systems-biology-led pathways, developed by Dr. Hai Fang and towards which I have made some contribution (manuscript in preparation). ATAC-seq and ChIPm peaks were annotated with gene entities based on publicly available promoter-HiC data in 17 immune cell types \parencite{Javierre2016}. The interactions were weighted based on the number of cell types for which the same bait-target interaction was reported as well as on the confidence of each of those interaction measured by the CHiCAGO score. This approach integrates the knowledge in the field about regulatory regions affecting the expression of distal genes and was not biased by the physical vicinity to a gene when annotating genomic intervals located at intergenic regions or gene desserts.For visualisation, manually curated KEGG pathways including all the genes for each gene family were colored based on the fold-change from the corresponding differential analysis and highlighted in bold when passing the FDR threshold for significance.

 \subsection{Enrichment analysis for genomic annotation features}
-Includes the ATAC all peaks enrichment for eQTLs 
-ATAC all peaks enrichment for GWAS
-ATAC for TFBS, chromatin annotation segments
GWAS with eQTL and DOCs using xLDenricher with co-localisation and permutation analysis 20,000


\subsection{Mass cytometry data analysis}
Mass cytometry analysis was performed by Dr Nicole Yager. Mean expression of cytokine release was calculated following unsupervised clustering analysis guided by a workflow similar to Nowicka \textit{et al.} with minor modifications \parencite{Nowicka 2017}. For each population of cells, cytokine production in SF and PB was calculated as the difference in the mean signal intensity between the 0h and 6h aliquots. Mean was chosen due to the extremely small changes observed when calculating median intensities. The percentage of cytokine release was calculated following manual gating for the CD14$^+$ population based on surface marker expression.  The percentage of TNF-$\alpha$ positive staining cells were calculated between the 0h and the 6h samples within each tissue.


\subsection{Statistical fine-mapping}


Fine mapping of the psoriasis and PsA GWAS signals was performed using a Bayesian approach, aiming to overcome the incomplete coverage of the genotyping arrays and the hundreds of associations per locus due to the LD structure of the genome. Fine-mapping was conducted using two different strategies upon availability of summary statistics or genotyping data from the psoriasis and PsA Immunochip GWAS studies, respectively. Both strategies include the same main steps of imputation, association testing and calculation of PP and were implemented in collaboration with Dr Adri\'{a}n Cort\'{e}s.

\subsubsection{PsA fine-mapping using Immunochip genotyping data}
Fine-mapping was performed for a subset of twenty-two non-MHC PsA Immunochip GWAS susceptibility loci previously selected by Bowes \textit{et al.} based on the lead SNP FDR$<$10$^-4$ and a marker density in the region of at least 100 SNPs.  The analysis was performed using 1,103 patients and 8,900 controls from the Bowes {et al.} PsA Immunochip UK cohort. Access to the data post-quality control was kindly provided by Dr Anne Baton (Manchester University). PCA analysis was performed using only pruned SNPs with flashpca \parencite{Abraham2014} and the calculated PCs were used as covariates into the association analysis to correct for population stratification. 

For each of the fine-mapped loci a 2Mb window from the lead SNP was defined and SNPs extracted from the data using PLINK 1.9 \parencite{Chang2015}. Phasing of the genotype data from that region was performed with SHAPEIT \parencite{Delaneau2012} and afterward used to impute missing genotypes with IMPUTE2 \parencite{Howie2009} and the 1000 Genomes Project Version 3 as the reference panel (October 2015 release). SNPs for which imputation was not successful in at least 70\% of the samples (info-score$<$0.7) were filtered out using QCtool and removed for downstream analysis to avoid confounders in the association analysis.

The association and conditional analysis was conducted using a Bayesian additive model implemented with SNPTEST and the PCs previously calculated as covariates to regress in the model \parencite{Burton2007}. Approximated Bayes Factor (ABF) was calculated for the lead signal and step-wise conditional analysis was performed if ABF$>$4. Credible sets of SNPs containing the variants more likely to explain 50\% and 90\% of that association were provided for each of the signals in the locus, along with their corresponding posterior probabilities (PP) as further detailed in Bunt \textit{et al.}, 2015.



\subsubsection{Psoriasis fine-mapping using Immunochip summary statistics data}
In the case of the psoriasis fine mapping was performed for X of the risk loci reported by the psoriasis Immunochip GWAS study from Tsoi \textit{et al.}, 2012. To conduct fine mapping only access to the summary statistics of the GAPC Immunochip cohort (2,997 cases and 9,183 controls) from Tsoi \textit{et al.}, 2012  was available. The summary statistics file included the pval and the OR calculated for each of the genotyped SNPs using a logarithmic regression model and correcting for ten principal components. Due to the restrained accessibility to genotyping data for ethical reasons, statistical methods have been developed to perform imputation of the summary statistics instead of the actual genotype for missing SNPs. In this case, the statistic z-scores from the genotyped SNPs were used by DIST \parencite{Lee2013} to impute the z-scores for allele 1 of the missing SNPs based on the correlation in linkage disequilibrium (r$^2$) from the 1000 Genome Project Version 3. Imputation was performed genome-wide for all the twenty-two autosomes and the results were filtered based on the quality of the imputation using threshold score$>$ 0.8.

After imputation of the summary statistics, association analysis and calculation of the ABF were performed using Wakefield approximation for a 2Mb window around the GWAS lead SNP of each locus of interest. This approximation was applied under the priors of  (1) normally distributed OR with mean and variance (v$^$2); (2) the greater the variance the bigger the size effects will be obtained; (3) mean is 0 and variance is fixed to 0.2 (accepted variance for GWAS studies). In this approach ABF was calculated using effect size (\beta) and standard error (SE) derived from the variance formula. $\beta$ was calculated using the z-scores of each of the interrogated SNPs as: $\beta$=z-score*SE. It is important to note that step-wise conditional analysis is not performed when using summary statistics imputation. Similarly to the genotyping fine-mapping approach, PP for the SNPs in a particular window (signal) were calculated and ranked to set the threshold of the 50 and 90 \% credible set of SNPs. 


