\begin{titlepage}
   \centering
   \begin{figure}
      \centering
%      \epsfig{file=oxford_logo.eps,width=\textwidth}
      \includegraphics[scale=0.4]{oxford_logo-eps-converted-to.pdf}
   \end{figure}
   {\LARGE{\textbf{Functional genomics of psoriasis}}}\\ 
    \vspace{2cm}
   {\Large{Alicia Lled\'{o} Lara}}\\
   {\Large{Hertford College}}\\
   {\Large{University of Oxford}}\\
   \vspace{2cm}   
   {\Large{\textit{A thesis submitted in partial \\ fulfilment of the requirements for the degree of\\ Doctor of Philosophy}}}\\
   {\Large{\textit{Hilary Term, 2019}}}
\end{titlepage}

\newpage
\chapter*{Abstract} %Creates chapter title but doesn't number
\addcontentsline{toc}{chapter}{\numberline{}Abstract} %Add abstract to table of contents
\thispagestyle{plain}
%\pagenumbering{gobble}
%\thispagestyle{empty}
\pagenumbering{roman} \setcounter{page}{1}

%\vspace*{-1.5cm}
\begin{center}
{{\bf Functional genomics of psoriasis}} \\
%
\end{center}
%\vspace*{-0.5cm}
\begin{center}

{Alicia Lled\'{o} Lara, Hertford College, Hilary Term 2019}\\
%\end{center}
%\vspace*{-0.5cm}
%\begin{center}
{A thesis submitted in partial fulfilment of the requirements
for the degree of Doctor of Philosophy of the University of Oxford} \\
%\end{center}
%\vspace*{-0.5cm}
%\begin{center}
\end{center}
%\vspace*{-0.5cm}

%\begin{center}
%{{\large \bf Abstract}}
%\end{center}

\noindent
This is my abstract... \\




\newpage
\chapter*{Acknowledgements}
\addcontentsline{toc}{chapter}{\numberline{}Acknowledgements}
\thispagestyle{plain}
%\pagenumbering{gobble}
%\thispagestyle{empty}
%\begin{center}
%{{\large \bf Acknowledgments}}
%\end{center}
\noindent
%
Thank you, thank you, thank you.

\newpage
\chapter*{Declarations}
\addcontentsline{toc}{chapter}{\numberline{}Declarations}
%\pagenumbering{gobble}
%\thispagestyle{empty}
\thispagestyle{plain}
%\begin{center}
%{{\large \bf Declarations}}
%\end{center}
\noindent
I declare that unless otherwise stated, all work presented in this thesis is my own. Some aspects of the thesis were a collaboration, with some of the work conducted with or by others.

All the healthy volunteers and psoriasis patientsâ€™ samples were collected by myself and processing was part of a collaborative effort with past and current lab members Dr Anna Sanniti, Dr Andrew Brown and Giuseppe Scozzafava. The psoriatic arthritis samples processed for ATAC, qPCR array and mass cytometry were part of the Immune Function in Inflammatory Arthritis (IFIA) study established in 2006 and sample collection was a collaborative effort with Dr Hussein Al-Mossawi and Dr Nicole Yager. 

The fixation protocol for sorted primary cells using DSP was optimised by Moustafa Attar. RNA extraction, ATAC and ChIPm processing for the healthy controls and psoriasis cohorts was carried out together with the ankylosing spondylitis samples in collaboration with Dr Anna Sanniti and Dr Andrew Brown. Advice for ATAC and ChIPm library indexing and sequencing was provided by Amy Trebes. RNA-seq and 10X Genomics technology Chromium single cell 3' expression library preparations and sequencing together with ATAC and ChIPm sequencing were performed by Oxford Genomics Centre at the Wellcome Centre for Human Genetics. Processing of the qPCR array and mass cytometry samples was conducted by UCB and measurement of synovial fluid cytokine and chemokine abundance was carried out by collaborators in Basle.


Regarding analysis, mass cytometry data was analysed by Dr Nicole Yager. ATAC and ChIPm NGS data processing was conducted using in house pipelines developed by Dr Gabriele Migliorini towards which I actively contributed by performing literature review analysis of the most appropriate methods, facilitating code for some of the parts and performing additional analysis to test and validate some of the tools and approaches. Te peak filtering strategy for ATAC using IDR was proposed and implemented by Dr Gabriele Migliorini and I conducted additional analysis to validate it. The strategy to perform filtering of chromatin accessible regions based on an empirical cut-off to remove excessive noise was developed and implemented together with Dr Hai Fang.RNA-seq NGS data processing was performed using the in-house pipeline developed by Dr Katie Burnham. All the resources for fine-mapping analysis using genotyping or summary statistics data were provided by Dr Adri \'{a}n Cort \'{e}s. General advice for analysis of different datasets were provided by Dr Silvia Salatino, Dr Hai Fang, Dr Katie Burnham, Dr Gabriele Migliorini, Dr Adri \'{a}n Cort \'{e}s and Enrique V \'{a}quez de Luis. The script to calculate enrichment across TSS was provided by Dr Silvia Salatino (part of the Oxford Genomics Centre resources). The function for colour-coding KEGG pathways based on gene expression data was developed by Dr Hai Fang and I contributed together with Dr Anna Sanniti to manual curation of the pathways.   


\newpage
\chapter*{Submitted Abstracts}
\addcontentsline{toc}{chapter}{\numberline{}Submitted Abstracts}
\thispagestyle{plain} %Removes the header
%\begin{center}
%{{\large \bf Associated publications}}
%\end{center}
\noindent

\noindent
\textbf{Title}
\hfill \textbf{Year}\\
Authors\\

\newpage
%\chapter*{Associated Publications}
%\addcontentsline{toc}{chapter}{\numberline{}Associated Publications}
%%\pagenumbering{gobble}
%%\thispagestyle{empty}
%\thispagestyle{plain} %Removes the header
%%\begin{center}
%%{{\large \bf Associated publications}}
%%\end{center}
%\noindent
%\textbf{Title}\\
%Journal\\
%Authors\\

\begingroup
\let\clearpage\relax
\chapter*{Other Publications}
\noindent
\textbf{Title}\\
Journal\\
Authors\\

\endgroup


\newpage
\addcontentsline{toc}{chapter}{\numberline{}Contents}
\tableofcontents

\newpage
\listoffigures
\addcontentsline{toc}{chapter}{\numberline{}List of Figures}

\newpage
\listoftables
\addcontentsline{toc}{chapter}{\numberline{}List of Tables}



\chapter*{Abbreviations}
\addcontentsline{toc}{chapter}{\numberline{}Abbreviations}

\begin{table}[H]
\singlespacing
  \centering
   \begin{tabular}{@{}m{2.5cm}m{10cm}@{}}
	  \textbf{Abbreviation} & Definition \\
		\textbf{Ab} & Antibody\\
		\textbf{ATAC-seq} & \\
		\textbf{Atopic dermatitis} & AD \\
		\textbf{ChIPm} &  \\
		\textbf{CLE} & cutaneous lupus erythematosus \\
		\textbf{DMARDs} & disease-modifying antirheumatic drugs \\
		\textbf{Fast-ATAC} & \\
		\textbf{IDR} & \\
		\textbf{GWAS} & Genome-wide association studies\\
		\textbf{KC} & Keratinocytes \\
		\textbf{NSAID} & nonsteroidal antiinflammatory drug \\
		\textbf{Omni-ATAC} & \\
		\textbf{PCA} & \\
		\textbf{PI} & Protein inhibitor \\
		\textbf{PsA} &  \\
		\textbf{QC} & \\
		\textbf{qPCR} & quantitative polymerase chain reaction \\
		\textbf{RA} & Rheumatoid arthritis \\
		\textbf{ROS} & Reactive oxygen species \\
		\textbf{SDS} & Sodium dodecyl sulfate \\
		\textbf{SF} & Synovial fluid\\
    \end{tabular}%
\end{table}%
%
