\chapter{General discussion}
\label{ch:Discussion}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
In this thesis, I have characterised, for the first time, cell type and context specific differences in chromatin accessibility and gene expression between patients and controls, and across tissues, in psoriasis and PsA. This required the establishment of methodological and analytical pipelines for ATAC and assessment of the effect that methods used to preserve clinical samples could have on the chromatin landscape. The use of low-input epigenetic techniques together with gene expression, single-cell analysis and the incorporation of GWAS fine-mapping data provides an integrated understanding of previously reported finding in psoriasis and PsA, adding cell and context specificity in physiological conditions. Although the results presented here have not unveiled new pathophysiological mechanisms, they represent a unique resource establishing the epigenetic and transcriptomic profiles of relevant cells in psoriasis and PsA, that could be further expanded and investigated at the functional level. 

This chapter will give an overview of the key findings of this thesis in relationship to the general aims and discuss the future work required to maximise the value of this data and its future potential clinical application.



\section{The importance of cell type and context specificity in disease}
The importance of cell type and context specificity in understanding regulation of gene expression has been widely demonstrated \parencite{Fairfax2012,Fairfax2014}. 
%This is particularly relevant in the investigation and understanding of complex chronic inflammatory diseases such as psoriasis and PsA, with implication of a variety of cell types and systemic and tissue specific features. 
In psoriasis and PsA, the enrichment of prioritised GWAS SNPs in putative enhancers and regulatory regions has highlighted various immune and non-immune cell types and tissues, including Th-1, Th17, memory  CD4$^+$ and CD8$^+$ cells, monocytes, keratinocytes, B cells and adipose tissue \parencite{Farh2015,Tsoi2017,Bowes2015,Ellinghaus2016,Lin2018}.

In this thesis, due to logistic and budget constraints, only a number pathophysiologically relevant cell types were investigated in psoriasis (CD14$^+$ monocytes, total CD4$^+$ and CD8$^+$ T cells and CD19$^+$ B cells) and PsA (CD14$^+$ monocytes, mCD4$^+$ and mCD8$^+$ T cells and NK cells). The variation in the number of differential targets and enriched pathways in chromatin accessibility and gene expression across cell types supports the need to expand this study to fully understand disease pathophysiology and aetiology. Of particular interest would be investigating plasmatocytoid DCs, key players in the skin inflammatory process that have been shown to be predominant at the psoriasis lesional skin, and for which psoriasis GWAS SNPs have shown context specific effects in their regulation of gene expression \parencite{Lee2014,Cheng2018}. 

Moreover, the epigenetic and transcriptional changes observed here between synovial fluid and peripheral blood of PsA patients demonstrates the importance of the environment at the site of inflammation to better understand disease pathophysiology, and highlights the need to isolate skin immune-infiltrated cells using FACS or single-cell technologies for further investigation \parencite{Ahn2017}

%The need of studying biological process and disease with tissue and cell type resolution is also being addressed by a number of consortiums which are expanding the generation of comprehensive epigenetic and transcriptomic maps in specific cell populations and conditions \parencite{roadmap, encode,blueprint} 




\section{The epigenetic landscape in psoriasis and PsA clinical samples: systemic vs affected tissue}
The epigenetic landscape in psoriasis revealed only marginal differences in chromatin accessibility and the active enhancer H3K27ac mark between patients and controls for peripheral blood cell types, with total CD8$^+$ cells demonstrating the largest number of differences. In contrast, the comparison of synovial fluid and peripheral blood in PsA revealed a larger number of differentially accessible regions in all the cell types, with CD14$^+$ monocytes showing greatest extent of changes in the chromatin accessibility landscape. Pathway enrichment analysis annotating the differential accessible regions with proximal genes showed that those changes were enriched for disease functional relevant processes. The contrasting results suggest that the physiological environment at the PsA inflamed joints drives chromatin accessibility changes of greater effect size than those observed in circulating cells between patients and control, and reinforce the need to study the chromatin accessibility landscape of skin in psoriasis. This is also in line with other studies where larger changes in chromatin accessibility or H3K27ac modifications have been found when directly interrogating the main pathological cell type or affected tissue \parencite{Scharer2016,Peeters2015,Wang2018,Liu2018}. 

Additional differences between circulating cells from psoriasis patients vs controls may also be unveiled by increasing sample size and studying more specific subsets within well defined cell populations, for example investigating memory and regulatory subsets within CD4$^+$ and CD8$^+$ populations. The success in assessing chromatin accessibility by Omni-ATAC in NHEKs (Chapter \ref{ch:Results1}) and the development of scATAC-seq opens the door to interrogate lesional skin keratinocytes and mapping the chromatin landscape of small numbers of infiltrated immune cells isolated from skin biopsies  \parencite{Buenrostro2015,Corces2017}. Lastly, in this thesis only one genomic region in CD8$^+$ cells showed paired changes in chromatin accessibility and H3K27ac between psoriasis patients and healthy controls, likely as a result of H3K27ac being only one of the diverse mechanisms involved in shaping chromatin accessibility. This may suggest that differences in this histone modification between patients and controls may not be relevant in driving the observed changes in chromatin accessibility. However, the overall small sample size in this study and the differences in sizes of the cohorts interrogated for ATAC and H3K27ac provided additional challenges to draw robust conclusion. 


%Since psoriasis has also shown evidence of systemic features, including changes in the proportion of certain cell types and levels of pro-inflammatory cytokines \parencite{Kagami2010,Sugiyama2005,Bai2018}, unveiling changes in the chromatin accessibility landscape between patients and controls in peripheral blood isolated cells may require increasing the sample size, as the effect size of those changes can be expected to be more modest compared to those expected when studying infiltrated immune cells at lesional inflamed skin from psoriasis patients. 


%Lastly, the method to perform differential chromatin accessibility analysis implemented this thesis is based on peak calling and has been adapted from RNA-seq methods with further optimisation, as shown in Chapter \ref{ch:Results1}. Other strategies not relying in peak calling (\textit{de novo}) used in ChIP-seq analysis could also be explored, but it is more sensitive to background noise and can increase the number of false positives \parencite{Shen2013}.




%The assessment of additional histone marks, together with an expanded cohort size will be required. This may have revealed larger differences in chromatin accessibility between psoriasis patients and controls compared to circulating immune cells, as in other studies performed in affected tissues \parencite{Scharer2016, Wang2018}. %Additionally, chromatin and transcriptomic profiles from skin infiltrated cells could be generated using FACS or single-cell technologies to better understand the changes in chromatin accessibility and gene expression driven by the inflammatory stimuli at the site of inflammation. Moreover, generating this data would also allow comparison to the profiles obtained in blood to better understand disease pathophysiology. 

%Importantly, direct interrogation of the main cell type or tissue affected by inflammation in those studies would partly explain the more profound changes observed in the chromatin landscape compared to my study. Additionally, some differences will be driven by genotype, and by the nature of co\subsubsection{Data integration}





\section{Cell type and tissue specific transcriptional profiles in clinical cohorts}

The majority of the transcriptional studies published in psoriasis and PsA have been conducted using PBMCs or SFMCs or in a particular cell type undergoing exogenous inflammatory stimuli. This has provided a very broad overview of dysregulated gene expression within mixed cell populations. Similarly, the study of lesional skin from psoriasis patients has been carried out in whole skin biopsies containing not only keratinocytes from the different layers of the skin and infiltrated immune cells, but also fibroblasts and adipose tissue. 

This thesis represents the first comprehensive study conducting a genome-wide transcriptomic analysis in four relevant immune cell types isolated from peripheral blood of patients and controls without additional stimulation. CD14$^+$ and CD8$^+$ were demonstrated as the main cell types undergoing dysregulation of gene expression, highlighting immune-relevant pathways with both up- and down-regulation of pro-inflammatory genes, and cell-type specific differentially expressed lncRNAs. Particularly unexpected was the up-regulation of two anti-inflammatory genes, \textit{TNFAIP3} and \textit{NFKBIA}, which are also psoriasis GWAS risk loci. \textit{TNFAIP3} dysregulation has also been reported in other chronic inflammatory diseases including UC, CD, cystic fibrosis and pediatric IBD, demonstrating disease specificity in the direction of the change \parencite{Jiang2012,Majumdar2017,Sahlol2019}. Interestingly, my data may also support a cell type and tissue specific effect in dysregulation of \textit{TNFAIP3} in psoriasis, since this gene is down-regulated in PBMCs and lesional skin from patients but shows up-regulated expression in peripheral blood CD4$^+$ and CD8$^+$ T cells \parencite{Jiang2012,Sahlol2019}.

The transcriptional study in epidermis sheets from lesional and uninvolved skin in patients highlighted larger changes in gene expression compared to the study in peripheral blood and the direction of the changes suggest a more pro-inflammatory signature, reinforcing again the role of the physiological inflammatory environment at the main disease affected tissue in driving functional changes. Moreover, the transcriptomic results also demonstrate the importance of aiming for pure cell populations, particularly of looking at purer keratinocyte samples from skin biopsies to unveil enrichment of not previously reported pathways. The use of epidermal sheets demonstrated enrichment of DEGs for the NOD-like pathways, which has not been reported by other studies using whole thickness skin biopsies, a finding that was further supported by a similar study published during this project \parencite{Tervaniemi2016}. 

The qPCR transcriptional study in three relevant cell types isolated synovial fluid and peripheral blood allowed, for the first time, comparison of cell-type specific dysregulation of inflammatory genes with changes in chromatin accessibility in PsA. My data showed the largest number of DEGs in CD14$^+$ monocytes, and similarly to the transcriptional results in skin, DEGs between synovial fluid and peripheral blood monocytes highlighted dysregulation of the NOD-like receptor signalling pathway by qPCR and scRNA-seq data, reinforcing the previously reported role of this pathway at the site of inflammation \parencite{McCormack2009}.

The use of scRNA-seq to further dissect cell heterogeneity within a population represents a groundbreaking advance in the study of disease. This is particularly important for the phenotypic chracterisation of very plastic cells such as monocytes, which can differentiate into macrophages, DCs and osteoclast upon different environmental stimuli \parencite{Qu2014,Rivollier2012}. In this thesis, a pilot exploratory study of subpopulations of monocytes in the synovium and peripheral blood of PsA was conducted for the first time and suggested the existence of functionally relevant subpopulations within the joints. Colleagues involved in this collaborative project are also exploring clonal expansion in T cells, with results suggesting relevant mCD8$^+$ clonal expansion in the joints compared to peripheral blood (manuscript in preparation). scRNA-seq has also been used to explore heterogeneity of pathphysiologically relevant cell populations in RA, MS and CD \parencite{Fumitaka2018,Zhang2018,Schafflick2018}. A recent study analysing healthy skin from different anatomical locations and lesional psoriatic epidermis has been published, revealing important differences across sites of sampling and showing expansion in the lesional epidermis of particular subpopulations also found in normal skin, further dissecting the aetiology of skin inflammation \parencite{Cheng2018}. Although this study described the composition of immune-infiltrated cells, further investigation and characterisation of these cells will be required. Recently, spatial transcriptomics has also yielded cell-type specific gene expression information from intact tissue sections of brain biopsies from MS patients or mice skin biopsies,  representing an exciting avenue in the study of PsA joints and psoriasis skin samples \parencite{Itoh2018,Joost2016}. 


%The scRNA-seq analysis in a different set of patients also confirmed the relevance in the dysregulation of NOD-like receptor signalling between the two tissues togeteher with the relevance of Monocytes plasticity has manifested in PsA where the greatest number of DEGs were found across PsA tissues and only moderate changes were observed when comparing this cell type from peripheral blood in PsA patients vs controls. Further investigation of scRNA-seq has revealed an CD14$^+$ monocytes IL7R$^+$ cluster, which predominates in synovial fluid  and has been confirmed at the protein level using FACS. In a publication currently under review, we have also demonstrated that characteristic markers of this discrete IL7R$^+$ cluster correlate with top DEGs from \textit{in vitro} IL-7 stimulated monocytes, reinforcing its role in PsA pathophysiology (Al-Mossawi \textit{et al.}, 2018 in revision).  %The large differences across monocytes from both tissues has also been observed in on going analysis by mass-cytometry and may explain, together with other factors, the failure of the current scRNA-seq clustering algorith to define subpopulations when combining the monocytes from both tissues. characterisation of the two monocytes populations was proven to be quite  to be quite different and . 



Although a systematic comparison between psoriasis and PsA is beyond the scope of this thesis, differences in gene expression for some of the targets were of interest. %For example \textit{S100A8} and \textit{S100A9} were up-regulation in lesional skin compared to uninvolved, but down-regulated in monocytes and mCD4$^+$ from synovial fluid compared to peripheral blood, in line with the strongest IL-17 signature reported in inflamed skin when compared to matched synovium from PsA patients \parencite{Belasco2015}. 
For example \textit{CCR10} was up-regulated  in total CD8$^+$ cells from psoriasis when compared to controls and also in mCD8$^+$ cells from synovial fluid in PsA. \textit{CXCL10} presented higher expression in PsA peripheral blood CD14$^+$ when compared to controls, showing a monocyte-specific up-regulated protein expression in synovial fluid vs matched peripheral blood. No changes were found in psoriasis vs healthy controls, in line with a recent conference abstract \parencite{Muntyanu2017}, and supporting the role of \textit{CXCL10} as potential PsA biomarker involved in the initiation and progression of joint inflammation previously hypothesised by another study \parencite{Abji2016}.




\section{Data integration in multi-omic studies}
The ability to generate different layers of data including chromatin accessibility, histone modifications, transcript quantification and protein expression represents a challenge when it comes to integration and interpretation.

%\subsubsection{Correlating epigenetic changes and gene expression}
One of the major difficulties in the interpretation of ATAC data relates to the challenge of linking accessible chromatin to the regulated target gene. Enhancers and other regulatory regions do not always exert their effect on the most proximal gene, and cell and context specific long-distance chromatin interactions play a crucial role in shaping the regulatory landscape \parencite{Smemo2014}. In this thesis annotation of the DARs is based on genes in physical proximity, in line with other current publications \parencite{Qu2015,Scharer2016,Wang2018} and could be refined by incorporating chromatin interaction, eRNA and additional functional data. %These challenges in annotating the chromatin landscape also limits the ability to perform robust correlations with changes in gene expression, particularly for regulatory elements distal from genes \parencite{Ackermann2016,Wang2018}. \
Recently, a publication by Liu and colleagues characterising the regulatory landscape of osteoarthritis used genome-wide integration of eRNA and chromatin segmentation maps to predict potential target genes of differentially accessible regions and correlate them with gene expression \parencite{Cao2017,Liu2018}.  Moreover, differential chromatin accessibility due to genetic variation has been shown to precede changes in transcript levels, and this should also be taken into account when trying to integrate epigenetic and transcriptional changes \parencite{Alasoo2018,Calderon2018}. Furthermore, the analysis of differential chromatin accessibility and histone modifications in psoriasis and PsA would benefit from incorporating genotyping data, which could eventually be expanded to perform chromatin accessibility QTL in disease \parencite{Mo2018,Alasoo2018}. This would allow the identification of accessible regions similar to the ATAC peak harbouring rs4672505 at the 2p15 GWAS locus, where chromatin accessibility appears to depend on the genotype in a cell-type specific manner. %This could be then expanded to the performance of chromatin accessibility eQTLs exploring the effect of genetic variants in healthy and disease state and also in the context of disease severity, similar to some studies using monocytes under different stimulations or eQTL studies comparing the effect of variants between juvenile idiopathic arthritis and IBD \parencite{Mo2018,Alasoo2018}. 

%\subsubsection{Correlation of transcriptomics and protein expression at the single-cell level}
The integration of scRNA-seq and mass cytometry data in the PsA pilot study was limited by the sample size, technical aspects and time scale, but provides a platform for future validation studies. To better understand the correspondence between transcriptional profiles and protein in discrete subpopulations of cells, a larger sample size plus a more systematic approach for data integration will be required. Currently Zhang and colleagues have presented the most exhaustive methodology to integrate bulk-RNA-seq, scRNA-seq, mass cytometry and flow cytometry into multi-modal transcriptomic and proteomics profiles \parencite{Zhang2018}. This strategy has revealed disease-specific functional expanded subpopulations amongst the most relevant cell types in RA pathophysiology. Moreover, generation of scATAC-seq data will allow further charactersisation and correlation of changes in chromatin accessibility and gene expression at the single-cell level \parencite{Duren2018}.

%A more systematic integrative approach should be implemented for the expanded cohort to establish appropriate correlation across datasets. Currently Zhang and colleagues have presented the most exhaustive methodology to integrate bulk-RNA-seq, scRNA-seq, mass cytometry and flow cytometry into multi-modal transcriptomic and proteomics profiles, but their work is still under peer review \parencite{Zhang2018}. This strategy has revealed disease-specific functional expanded subpopulations amongst the most relevant cell types in RA pathophysiology. Additionally, the correlation between bulk ATAC and scRNA-seq is clearly limited by the different scales of the two approaches. Therefore, generation of scATAC-seq data, identification of clusters based on chromatin profiling and appropriate methods for the overlap with scRNA-seq populations should be used to have a better understanding of the correlation between chromatin accessibility and gene expression at the single-cell level \parencite{Duren2018}.mplex diseases different patients have different genetic backgrounds, with some variants also shared with control individuals. Thus it may be necessary to study changes in chromatin accessibility in the context of genotype and under exogenous inflammatory stimuli that may manifest those differences \parencite{Alasoo2018,Calderon2018}.
%A more systematic integrative approach should be implemented for the expanded cohort to establish appropriate correlation across datasets. Currently Zhang and colleagues have presented the most exhaustive methodology to integrate bulk-RNA-seq, scRNA-seq, mass cytometry and flow cytometry into multi-modal transcriptomic and proteomics profiles, but their work is still under peer review \parencite{Zhang2018}. This strategy has revealed disease-specific functional expanded subpopulations amongst the most relevant cell types in RA pathophysiology. Additionally, the correlation between bulk ATAC and scRNA-seq is clearly limited by the different scales of the two approaches. Therefore, generation of scATAC-seq data, identification of clusters based on chromatin profiling and appropriate methods for the overlap with scRNA-seq populations should be used to have a better understanding of the correlation between chromatin accessibility and gene expression at the single-cell level \parencite{Duren2018}.




\section{GWAS fine-mapping and integration with patients-derived epigenetic data}
Statistical fine-mapping is a widely accepted strategy for refining putative causal SNPs for functional follow-up. Fine-mapping using Bayesian methods have been conducted using Immunochip summary statistics and genotyping data for psoriasis and PsA, respectively. In both cases the power of the analysis was limited by the incomplete cohort size, which explained the variation of results when compared to the fine-mapping conducted by Bowes and colleagues in PsA \parencite{Bowes2015}. Further refinement of the credible set of SNPs was achieved by integration of the in-house ATAC data generated in psoriasis and PsA samples and highlighted two putative causal variants at the \textit{B3GNT2} and \textit{RUNX3} loci in psoriasis and PsA, respectively. The overlap of rs11249213 with a mCD8$^+$ synovial fluid open DAR which was significant for FDR$<$0.05 but not FDR$<$0.01, suggests the potential for finding additional overlaps with DARs of smaller effect size that may become significant in larger studies.

Additionally, the step-wise Bayesian fine-mapping strategy implemented here performs under the assumption of only one SNP being causal for a particular association \parencite{Maller2012,Bunt2015}. In contrast, other methods such as Bayesian evolutionary stochastic search allow to test for haplotype models where several variants may be driving the association \parencite{Wallace2015} and should be take into account for further analysis . 

%highlighting the potential of identifying more overlaps at differential regions with smaller effect size by lowering the threshold for significance and increasing the sample size. Integration of ATAC data from the psoriasis and control cohort at the chr2p15 showed a CD8$^+$ specific overlap for rs4672505, with loss of chromatin accessibility correlating with the risk allele, and suggesting a putative causal role through regulation of \textit{B3GNT2} expression in this cell type. Overall, this is the first time that patient-derived epigenetic data has been integrated with psoriasis and PsA fine-mapped GWAS variants, showing interesting results for some of the loci that could be followed-up.

Notably, in this thesis overlap of psoriasis and PsA GWAS fine-mapped SNPs with significant eQTL SNPs (FDR$<$0.05) from relevant cell types or tissues was reported. These overlap were not necessarily between the lead SNPs of each of the signals, and colocalisation analysis using tools such as Coloc \parencite{Giambartolomei2014} could be used to formally determine whether the two fine-mapping and eQTL signals are the same and thus increase evidence of non-coding GWAS SNPs regulating expression of a particular gene.



\section{Clinical relevance and translation}
Technological advances in sequencing and the latest low-input and time-efficient epigenetic techniques, such as ATAC, enable the implementation of molecular tools based on gene expression and chromatin accessibility in a clinical setting. Genotyping, epigenetic and expression data could be used in the frame of personalised medicine to build disease, cell type, genotype and context-specific chromatin segmentation maps to help in diagnosis, patient stratification, treatment decisions and drug design. Nevertheless, clinical translation of findings are challenging due to variability in the results across studies, influenced by sample size, age, sex and patient heterogeneity \parencite{Shen-Orr2013}. Interindividual heterogeneity has been assessed at the chromatin accessibility level in T cells, where gender was the main driver of variation and variable regions presented significant enrichment at autoimmune disease loci \parencite{Qu2015}.

In cancer, gene expression profiling has revealed utility for diagnosis and chromatin accessibility has also been investigated in chronic lymphocytic leukemia, amongst others \parencite{Whitney2003,Rendeiro2016}. In inflammatory diseases such as eosinophilic esophagitis, the expression of 94 genes in esophageal biopsies has been successfully used to diagnose disease and assess treatment response \parencite{Dellon2018}. Currently, no molecular diagnosis based on gene expression or epigenetic profiles are formally used for psoriasis or PsA. A microarray study using PBMCs identified levels of \textit{MAP3K3} and \textit{CACNA1S} expression as the best combination of genes to correctly classify PsA and RA patients \parencite{Batliwalla2005}. Expression levels in skin biopsies of a set of seven genes, including \textit{CXCL8}, \textit{CXCL10} and \textit{CCL17}, appropriately discriminated psoriasis from atopic dermatitis, allergic contact dermatistis and irritant contact dermatitis \parencite{Zeeuwen2008}. A pilot metabolomic study showed that PsA patients had decreased level of alpha ketoglutaric acid and an increased level of lignoceric acid which could potentially be used to distinguish between the two pathologies in blood sera \parencite{Armstrong2014}. Further analysis and validation of the results generated in this thesis will contribute to progress towards the clinical translation of epigenetic profiles, transcriptomics and protein expression in psoriasis and PsA.

%expression of \textit{APO1} and \textit{DUOX2} together with microbiome composition has been used to differentiate between CD and UC patients and gene signatures have also been established to discriminate between high and low inflammatory profiles within each group \parencite{Haberman2014,Montero-Melendez2013}. Similarly, 



\section{Future work}
\subsubsection{Extended recruitment and sample types}

As previously mentioned, there are a number of factors limiting the identification of differences in chromatin accessibility and gene expression that are disease-specific and not due to general inflammation. The small sample size and unavailability of samples from the most relevant affected tissue are two major limitations. Additionally, patient heterogeneity and uncertainty due to conversion of psoriasis patients into PsA also needs to be taken into account to interpret these results.

In order to overcome limitations due to sample size, recruitment of additional psoriasis and PsA patients samples would need to be conducted. Expansion of the psoriasis recruitment should include peripheral blood, lesional and uninvolved skin from patients not undergoing biological treatment and also matched samples from those individuals undergoing biological therapy such as TNFi. Regarding skin samples, paired scATAC-seq and scRNA-seq could be performed on the infiltrated immune cell types at the site of inflammation and compared to their counterparts in peripheral blood. PsA na\'{i}ve-treated patient recruitment would also aim to include skin biopsies and samples following different types of biologic therapies. Moreover, peripheral blood and synovial fluid from healthy controls and individuals suffering from arthritic pathologies, such as RA and osteoarthritis, could also be included to enable comprison between chronic inflammation and PsA specific features. In the extended recruitment, unification in the methodology of processing samples will also be implemented to enable comparison across psoriasis and PsA patients. Lastly, additional clinical data and follow-up evaluation of the psoriasis patients by a rheumatologist would reinforce appropriate classification of patients as psoriasis non-converters into PsA.

%Additional clinical data to maybe be able to better determine the status from the psoriasis patients
%Recruitment of additional patients would allow validation of the findings described in this chapter. Genotyping data would permit the study of chromatin accessibility in a genotype-specific manner, using the current samples with prospective integration of chromatin conformation data \parencite{Kumasaka2018}. Importantly, this will enable exploration of changes in chromatin accessibility at GWAS loci in combination with fine-mapping, similarly to the chr2p15 locus analysed in this chapter. %Furthermore, new sample recruitment could be used to study chromatin accessibility and gene expression in additional cell populations sorted by FACS and also to include \textit{in vitro} stimulations. Overall, this strategy would allow better characterisation of the differences and similarities between patients and controls in context-specific regulatory elements \textit{in vivo} and \textit{in vitro} \parencite{Peeters2015}.



\subsubsection{Further exploration of chromatin accessibility and histones modification analysis}
The successful implementation of the optimised Omni-ATAC protocol in NHEKs (Chapter \ref{ch:Results3}) will allow future generation of accessibility data in keratinocytes isolated from lesional and uninvolved skin biopsies from psoriasis and PsA samples. Moreover, ChIPm would enable interrogation of additional histone marks, including H3K4me1 (enhancer) and H3K4me3 (promoter), RNA-pol II and TFs such as CTCF. Performing ATAC and ChIPm in the same relevant cell types isolated from patients and controls, followed by stimulation could focus the analysis on a set of promoters and regulatory regions undergoing modifications in inflammation \parencite{Peeters2015}. %This could be used to define a set of "immune enhancers" to restrict the analysis, as in previous studies interrogating H3K27ac in juvenile idiopathic arthritis \parencite{Peeters2015}, and increase the power of the study even with a lower number of samples.

In terms of analysis, \textit{de novo} methods not based on peak calling could be used to further explore differences in chromatin accessibility and histone marks \parencite{Shen2013}. Moreover, genotyping of all the subjects would allow the exploration of changes in chromatin and histone modifications in the context of genotype, %This will involve further exploration of variable peaks across individuals, particularly those in the vicinity of psoriasis and PsA GWAS loci, which may show a genotype-dependent effect in terms of chromatin accessibility or histone modifications enrichment, 
similarly to the chr2p15 example illustrated for psoriasis. Furthermore, this type of analysis could be expanded to generate chromatin accessibility and histone modification QTLs in a tissue, cell type and disease status specific manner.

Interpretation of genomic regions data, importantly ATAC will be improved using a tool currently under development known as Atlas and Analysis of Pathways (A2), created by Dr Hai Fang in collaboration with Dr Anna Sanniti and myself. This tool will incorporate pathway-centric interpretation of genomic regions through identification of crosslinked genes based on experimentally-derived evidence, including eQTLs, chromatin interaction or enhancer-promoter co-expression. This analysis will improve and refine the interpretation of the differential chromatin accessibility analysis in PsA and any genomic-region based data.



\subsubsection{Additional gene expression analysis and allele specific expression}
In order to have a better understanding of the genome-wide changes in gene expression between PsA immune cells from synovial fluid and peripheral blood, RNA-seq will need to be performed in the same cell types currently interrogated by qPCR, and also in additional cell populations of interest. Allele-specific expression (ASE) analysis could be performed to assess the \textit{cis} effects of genetic variants in gene expression and removing confounders such as \textit{trans}-acting environmental effects. Moreover, miRNAs could also be interrogated in future to further investigate the role of non-coding RNAs in disease.%will the lncRNAs from the psoriasis differential analysis and from future PsA RNA-seq data could be further investigated by integrating expression of their experimentally validated targets and identification of networks of co-regulated transcripts, similarly to the analysis performed by Dolcino and colleagues \parencite{Dolcino2018}. 

\subsubsection{Data integration}
 Integration of the histone marks and chromatin accessibility changes in a larger cohort may reveal co-regulated changes in a disease or genotype specific manner, revealing additional information about the mechanism of action of risk variants. Additionally, ASE data could also be integrated with the cell-type specific epigenetic maps from the same individuals and fine-mapped analysis to help informing the mechanisms of action of putative causal variants. Better understanding of disease pathophysiology and genetic associations would be aided by incorporating psoriatic skin methylation data and other publicly available chromatin accessibility QTLs and methyl-QTLs generated in different tissues \parencite{Chandra2018,Pelikan2018,Kumasaka2015,Kumasaka2018,Chen2016,}.
Integration of single-cell RNA-seq and mass cytometry data will also benefit from improvements in the methods for validating cluster identities, one of the most challenging aspects in single-cell analysis \parencite{Kiselev2019}. Additionally, systematic integration of single-cell mass-cytometry and RNA-seq data should be considered to find correlation between relevant subpopulations in each assay. This could be achieved by algorithms based on Canonical Correlation Analysis that have been implemented in the study of RA \parencite{Zhang2018}.


\subsubsection{Functional follow-up}
The epigenetic and gene expression data generated in this thesis, in addition to publicly available functional data, has pritoritised variants at GWAS loci that merit further investigation. One such example is rs4672505, which is hypothesised to be regulating \textit{B3GNT3} expression in CD8$^+$ T cells. Genomic-editing using CRISPR-cas9 \parencite{Ding2013} could be used to delete the ATAC peak colocalising with rs4672505 in CD8$^+$ cells and determine the effect in the regulation of \textit{B3GNT2} expression in homeostasis and also under anti-CD3 and anti-CD28 stimulation. %This would allow the definition of an enhancer at this location and would enable to investigate the effect of different haplotypes by selectively modifying the genotype at particular sites of the genome. 
Chromatin interaction experiments using capture-C could also be conducted in untreated and stimulated cells to assess the effect of rs4672505 genotype in the physical interaction with the \textit{B3GTN2} promoter seen by Javierre and colleagues \parencite{Javierre2016}. Additionally, molecular assays to interrogate allelic differences in affinity for TF binding, including electrophoretic mobility shift assay (EMSA) and also ChIP using antibodies for TF of interest in CD8$^+$ cells harbouring different rs4672505 genotypes could be performed \parencite{Vernes2007}.% in the case of rs4672505 to test differential affinity of \textit{STAT1} for each of the alleles.
 In terms of genome-wide screening, self-transcribing active regulatory region sequencing (STARR-seq) \parencite{Arnold2013} could be implemented to create a genome-wide quantitative enhancer map that ccould be colocalised with epigenetic data and non-coding fine-mapped SNPs.


%\textit{In vitro} assays to investigate the effect of genetic variants in regulating gene expression, involve for example transfection of constructs containing the promoter or enhancer element to test followed by luciferase expression \parencite{Niimi2002}. Other molecular assays to interrogate allelic differences in affinity for TF binding include electrophoretic mobility shift assay (EMSA) and ChIP using Ab for the particular TF of interest \parencite{Vernes2007}. The need to perform these assays at a genome-wide scale has yielded to development of high-throughput technologies, such as massively parallel reporter assays (MPRAs), which test putative enhancers and the effect of genetic variability in their functionality \parencite{Kheradpour2013}. In addition to this, mass spectrometry (MS) techniques have been used to perform allele-specific quantitative proteomics and have revealed allele-dependent binding of TF and co-regulators at the T2D \textit{PPARG} GWAS locus \parencite{Lee2017}. \textit{In vivo} validation has traditionally involved the use of mice models, including knock-outs for the potentially pathological genes or the regulatory elements containing GWAS prioritised variants. Nevertheless, the use of mice models to study human genotype-phenotype relationships has shown to have limitations that need to be taken into account when interpreting the results \parencite{Ermann2012}. 
%
%Both, \textit{in vitro} and \textit{in vivo}, models for functional studies have benefited from the development a the genome-editing technology known as clustered regularly interspaced short palindromic repeats (CRISPR/Cas) \parencite{Cong2013}. CRISPR/cas enables monoallelic and biallelic modifications of primary cells and embrionic stem cells (ESCs) for the particular SNP or region of interest. The limitations of CRISPR to edit certain primary cells is being overcome by the use of human induced pluripotent stem cells (hiPSCs), which can undergo terminal differentiation into the cell type of interest after CRISPR modification \parencite{Ding2013}. 
%%For example, neurons differentiated from CRISPR/cas edited iPSCs have been used in schizophrenia to functionally dissect a risk haplotype in the promoter of \textit{MIR137} \parencite{Forrest2017}.  %which exerts in reducing chromatin accessibility at the \textit{MIR137} promoter and expression responsible for neural dendritic complexity and synapse maturation \parencite{Forrest2017}. 
%%Moreover, CRISPR/cas has also been used for high-throughput interference screens (CRISPRi) to discover regulatory elements and identify their target genes by altering chromatin state at particular locus \parenbcite{Fulco2016}. Similarly, CRISPR activation (CRISPRa) assays have been used to identify stimulus-responsive enhancers independently of stimulus exposure, which represents allows simplification of the experimental designs \parencite{Simeonov2017}. Both approaches have been used in combination with HiChIP for validating linkage of non-coding variants to putative gene targets \parencite{Mumbach2017}.


\section{Concluding remarks}
The overall aim of this thesis was to investigate the epigenetic regulatory landscape and the transcriptional profile of psoriasis and PsA patients in a cell type and tissue specific manner to further help understanding of disease mechanism, requiring establishment of methodological techniques and analytical pipelines to interrogate chromatin accessibility. The data generated in this thesis has enabled a genome-wide comparison of the chromatin landscape and gene expression in disease vs healthy controls and also at the site of inflammation. Integration with fine-mapping GWAS and other publicly available datasets has further refined the number of putative causal variants for functional follow-up and reinforced the importance of integrating multiple cell type and context specific datasets to interpret GWAS results. As the datasets generated here continue to be expanded and experimental and analytical techniques are further refined, the understanding of disease mechanisms governing psorasis and PsA in important cell types and states will no doubt continue.



%The overall aim of this thesis was to establish a methodological and analytical pipeline to interrogate chromatin accessibility and to start investigating psoriasis and PsA disease mechanisms at the epigenetic and transcriptional level in a cell type and tissue specific manner. The data generated in this thesis has enabled a genome-wide comparison of the chromatin landscape and gene expression in disease vs healthy controls and also at the site of inflammation. Integration with fine-mapping GWAS and other publicly available datasets has further refined the number of putative causal variants for functional follow-up and reinforced the importance of integrating multiple cell type and context specific datasets to interpret GWAS results. As the datasets generated here continue to be expanded and experimental and analytical techniques are further refined, the understanding of disease mechanisms governing psorasis and PsA in important cell types and states will no doubt continue.




%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%



%Metabolomics in psoriasis and PsA
%https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4288418/


% Additional work for B3GNT2
%such as increasing the sample size to acquire more homozygous individuals for the risk allele, studying chromatin accessibility and \textit{B3GNT2} expression in relation to rs4672505 genotype in stimulated CD8$^+$ and performing genome editing to establish a causal relationship between the resik allele and altered gene regulation.


%Future work in psoriasis
%Additionally, chromatin and transcriptomic profiles from skin infiltrated cells could be generated using FACS or single-cell technologies to better understand the changes in chromatin accessibility and gene expression driven by the inflammatory stimuli at the site of inflammation. Moreover, generating this data would also allow comparison to the profiles obtained in blood to better understand disease pathophysiology. 

%Furthermore, new sample recruitment could be used to study chromatin accessibility and gene expression in additional cell populations sorted by FACS and also to include \textit{in vitro} stimulations. Overall, this strategy would allow better characterisation of the differences and similarities between patients and controls in context-specific regulatory elements \textit{in vivo} and \textit{in vitro} \parencite{Peeters2015}.

%ATAC enhancers to target gene
%These could involve a more systematic integration of available chromatin conformation data, eRNA FANTOM data and also use of analytical models and tools currently available or that may be further developed in the future to specifically address this challenge \parencite{Wang2016,Cao2018}.

%From PsA discussion
%Moreover, accessible chromatin is not a definitive marker for regulatory regions and mapping of histone marks such as H3K4me1 and H3K27ac together with eRNA quantification will help to refine the functional relevance of the identified DARs.  


%Limited sample size
%This results from difficulties of recruiting PsA patients na\'{i}ve for any treatment, the logistical difficulties to coordinate all of the techniques from the same sample, and the high cost of this approach. 

